### PR TITLE
Rewrote Getting Started & index file

### PR DESCRIPTION
**Pull request by AnnieLR**
Please read thru it all.
Not sure if the install section is correct. To Do task for me: create a separate more detailed doc for install & configure. 
Should we link from "Getting Started" to 'index' page w/ detailed "story of Stylable"?

### CODE REVIEW

File: site/docs/getting-started.md [hunk 1]
```diff
@@ -5,30 +5,38 @@ redirect_from: "docs/index.html"
...
+## How do I install
 
-At Wix, we agree. We love CSS. It's simple, declarative syntax that is native in browsers and is easily the fastest way to add styles to web pages and web apps. But when writing CSS that is scoped to individual components, developers have to maintain highly-specific selectors, using elaborate conventions to fake namespacing. Writing and maintanining CSS across large teams and large projects can be tricky.
+1. Install **Stylable** from our GitHub [repository](https://github.com/wix/stylable).
 
-We want to extend CSS so that it is easier to use in a component ecosystem, but without losing the aspects of CSS that make it great: declarative, familiar, static and fast. So we created **Stylable**.
+2. Install the **Stylable** [integrations package](https://github.com/wix/stylable-integration).
 
-**Styable** is a preprocessor that allows you to write style rules in CSS syntax, with some extensions that we believe adhere to the spirit of CSS.
+3. While **Stylable** can work with any type system, the most commonly used is React. Install the latest version.
```
Comment (arnonkehat) [inline site/docs/getting-started.md]: React is not a Type System. You can say:

"While **Stylable** can work with any component framework, here at Wix we use it with React."

File: site/docs/getting-started.md [hunk 2]
```diff
@@ -5,30 +5,38 @@ redirect_from: "docs/index.html"
...
+
+* **Remember** - Our [cheatsheet](../docs/getting-started/cheatsheet.md) helps you keep track of **Stylable** syntax.
 
-Viva CSS, and welcome **Stylable**. We hope you like it. Learn how to use it with step by step instructions and code examples.
 
 ## Components
 
-[Stylable-components](https://github.com/wix/stylable-components) - **Stylable**, fully-tested & strictly-typed component library based on React.
+**Stylable** includes a rich [component library](https://github.com/wix/stylable-components) for out-of-the-box use.
+**Stylable** components are a fully-tested & strictly-typed component library based on React.
```
Comment (arnonkehat) [inline site/docs/getting-started.md]: Replace with:

We recommend using our component library, [**stylable-components**](https://github.com/wix/stylable-components) . **Stylable Components** mare a fully-tested & strictly-typed React components

**Comment by brucelawson** [comment]
Not sure we need a 'story of Stylable' yet - be an interesting blogpost from Ido when we go public, though

**Comment by arnonkehat** [comment]
@AnnieLR another request: can we put a link to the doc center in the repo readme?

**Pull request merged**